interactive/forms.py

```diff
@@ -9,19 +9,8 @@ class Meta:
         fields = ["full_name", "email", "organisation", "job_title"]
 
 
-class AnalysisRequestForm(forms.ModelForm):
```

Comment:
Asking so that I can learn more Django best practice... 

* What's the advantage of moving to Form from Model Form?
* Why move the save into the view instead of the form? It seems like this is causing the view to do more work?

Reply:
ModelForms couple form validation with model creation.  They're a useful tool, particularly early on, but often bring complexity to a project when overused.  It's tricky to define a definitive cut off point from moving away from them.  

Here, I've switched because we're no longer using any fields from the model so the automated creation of fields, that a ModelForm brings us, is a no-op and removes much of the purpose of one.  Forms can't save models so pulling the save into the view was so we could create the object there.  This also makes the form a simple validation layer now.

We could take this further and wrap up the creation of an AnalysisRequest in either a classmethod (eg `AnalysisRequest.create()`) or even add in something like [CQRS](https://martinfowler.com/bliki/CQRS.html), but I didn't feel either added anything to the code at this point.

Reply:
Thank you! That makes a lot of sense.